dataactvalidator/config/sqlrules/a18_cross_file.sql

```diff
@@ -1,10 +1,26 @@
 SELECT
-    approp.row_number,
-    approp.tas,
-    approp.gross_outlay_amount_by_tas_cpe,
-    SUM(op.gross_outlay_amount_by_pro_cpe) as gross_outlay_amount_by_pro_cpe_sum
-FROM appropriation approp JOIN object_class_program_activity op
-ON approp.tas = op.tas AND approp.submission_id = op.submission_id
+	approp.row_number,
+	approp.allocation_transfer_agency,
+	approp.agency_identifier,
+	approp.beginning_period_of_availa,
+	approp.ending_period_of_availabil,
+	approp.availability_type_code,
+	approp.main_account_code,
+	approp.sub_account_code,
+	approp.gross_outlay_amount_by_tas_cpe,
```

Comment:
Why does this need to add a bunch of new fields in the `select` and `group by`? Generally, we shouldn't need to `group by` fields that aren't part of the `having` (and potentially `where`) clause. Is `approp.row_number` not unique within the table?